test/test.config.js

```diff
@@ -24,3 +26,21 @@ config.mongodb.dropCollections.collections = [];
 
 // allow self-signed certs in test framework
 config['https-agent'].rejectUnauthorized = false;
+
+// this needs to be true in order to set auth stuff
```

Comment:
This comment a little vague, can it be reworded or more specific? What is "auth stuff"? Maybe something like this?

```suggestion
// setting this automatically creates login sessions for the tests when new
// accounts are created
```